trackpy/feature.py

```diff
@@ -342,7 +344,7 @@ def _refine(raw_image, image, radius, coords, max_iterations,
         return np.column_stack([final_coords, mass, Rg, ecc, signal, raw_mass])
 
 
-def locate(raw_image, diameter, minmass=100., maxsize=None, separation=None,
+def locate(image, diameter, minmass=100., maxsize=None, separation=None,
```

Comment:
This was just a rebase error. Changing `image` back to `raw_image` causes all tests to pass.